DataFormats/CTPPSReco/interface/CTPPSPixelRecHit.h

```diff
@@ -0,0 +1,100 @@
+/*
+ *
+* This is a part of CTPPS offline software.
+* Author:
+*   Fabrizio Ferro (ferro@ge.infn.it)
+*
+*/
+
+#ifndef DataFormats_CTPPSReco_CTPPSPixelRecHit_H
+#define DataFormats_CTPPSReco_CTPPSPixelRecHit_H
+
+
+#include "DataFormats/GeometrySurface/interface/LocalError.h"
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+
+// Reconstructed hits in CTPPS Pixel detector
+
+class CTPPSPixelRecHit{
+
+public:
+
+  CTPPSPixelRecHit(){}
+CTPPSPixelRecHit(LocalPoint lp, LocalError le) : 
+  thePoint(lp), 
+    theError(le), 
+    isOnEdge_(false),
+    hasBadPixels_(false), 
+    spanTwoRocs_(false), 
+    minPixelRow_(0), minPixelCol_(0), 
+    clusterSize_(0), 
+    clusterSizeRow_(0), 
+    clusterSizeCol_(0)
+    {}
+
+CTPPSPixelRecHit(LocalPoint lp, LocalError le, bool edge, bool bad, bool rocs) : 
+  thePoint(lp), 
+    theError(le), 
+    isOnEdge_(edge),
+    hasBadPixels_(bad), 
+    spanTwoRocs_(rocs),  
+    minPixelRow_(0), 
+    minPixelCol_(0), 
+    clusterSize_(0), 
+    clusterSizeRow_(0), 
+    clusterSizeCol_(0)
+    {}
+
+CTPPSPixelRecHit(LocalPoint lp, LocalError le, bool edge, bool bad, bool rocs, 
+		 int minrow, int mincol, int size, int rowsize, int colsize) : 
+  thePoint(lp), 
+    theError(le), 
+    isOnEdge_(edge),hasBadPixels_(bad), 
+    spanTwoRocs_(rocs),  minPixelRow_(minrow), 
+    minPixelCol_(mincol), clusterSize_(size), 
+    clusterSizeRow_(rowsize), 
+    clusterSizeCol_(colsize)
+    {}
+
+  
+  inline LocalPoint getPoint() const { return thePoint;}
+  inline LocalError getError() const { return theError;}
+  
+  inline bool isOnEdge() const {return isOnEdge_;}
+  inline bool hasBadPixels() const {return hasBadPixels_;}
+  inline bool spanTwoRocs() const {return spanTwoRocs_;}
+
+  
+  inline unsigned int minPixelRow() const { return minPixelRow_;}
+  inline unsigned int minPixelCol() const { return minPixelCol_;}
+ 
+  inline unsigned int clusterSize() const { return clusterSize_;}
+  inline unsigned int clusterSizeRow() const { return clusterSizeRow_;}
+  inline unsigned int clusterSizeCol() const { return clusterSizeCol_;}
+
+private:
+
+  LocalPoint thePoint;
```

Comment:
thePoint_  (and  theError_), with the same trailing underscore as the other class member variables

Reply:
done